.github/workflows/ci_cd.yml

```diff
@@ -50,18 +51,18 @@ jobs:
         uses: actions/upload-artifact@v3
         with:
           name: documentation-pdf
-          path: _build/*.pdf
+          path: _build/{*.pdf,*.png}
           retention-days: 7
 
       - name: Prepare files for gh-pages deployment
-        if: github.ref == 'refs/heads/main'
+        #if: github.ref == 'refs/heads/main'
```

Comment:
```suggestion
```

Reply:
Let's try to avoid doing this in the future, please. There is a reason why we upload the documentation artifacts in HTML format as well. Do not commit to gh-pages branch unless it is merged to main. Let's follow proper software development procedures...

Reply:
Agreed @RobPasMue .